README.md

```diff
@@ -1,6 +1,33 @@
+# Azure Storage Blob SDK for Go
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud storage. 
 
-# Contributing
+This repository contains the open source Blob SDK for Go. Documentation and examples can be found [here](https://godoc.org/github.com/Azure/azure-storage-blob-go/2016-05-31/azblob).
 
+## Features
+* Blob Storage
+	* Create/Read/Update/Delete Block Blobs
+	* Create/Read/Update/Delete Page Blobs
+	* Create/Read/Update/Delete Append Blobs
+
+## Getting Started
+* If you don't already have it, install [the Go Programming runtime](https://golang.org/dl/)
+* Go get the SDK:
+
+```go get -u https://github.com/Azure/azure-storage-blob-go```
```

Comment:
This should be: go get github.com/Azure/azure-storage-blob-go/2016-05-31/azblob

Reply:
Done